src/routers/alpha-router/config.ts

```diff
@@ -30,6 +30,14 @@ export const DEFAULT_ROUTING_CONFIG_BY_CHAIN = (
           topNWithEachBaseToken: 3,
           topNWithBaseToken: 3,
         },
+        v4PoolSelection: {
+          topN: 2,
```

Comment:
i assume we are following v3?  how come v2 different?

Reply:
I think prior to launch Blast, we expected V2 pools to be more popular on Blast, because Blast stablecoin USDB and native ETH is a yield-bearing token, which means it works well on Uniswap V2 pools, but not v3 pools. Hence the search space on Blast v2 pools will be slightly larger than Blast V3 pools.

Blast is kinda unique case, since we don't see other Uniswap supported chains having yield-bearing native token and stablecoin.